src/components/MoviesList/MovieBox/MovieBox.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from 'prop-types';
```

Comment:
it's better to be consistent with a style of quotes 😄

Reply:
yes, sure)